tests/test_cli_data.py

```diff
@@ -26,45 +27,7 @@
 )
 
 
-@pytest.fixture(scope="session", autouse=True)
```

Comment:
we move this to tests.utils, so we can re-use this config block across all test modules